unsafe/src/main/java/org/apache/spark/unsafe/types/Interval.java

```diff
@@ -44,4 +44,9 @@ public boolean equals(Object other) {
   public int hashCode() {
     return 31 * months + (int) microseconds;
   }
+
+  @Override
+  public String toString() {
+    return "interval: " + months + " months and " + microseconds + " microseconds";
```

Comment:
can we make this better?

0. remove the : after interval
1. if months = 0, don't show the month part
2. ideally, show x days y seconds and such, instead of showing directly the number of ms